frontend/verification/lib/VerificationPage2.dart

```diff
@@ -0,0 +1,249 @@
+                  value: choice,
+                  child: Text(choice),
+                );
+              }).toList();
+            },
+          )
+        ],
+      ),
+      body: Center(
+        child: ListView(
+          padding: const EdgeInsets.all(8.0),
+          children: <Widget>[
+            Container(
+              height: 50.0,
+              child: const Center(
+                child: Text(
+                  'Verification details',
+                  style: TextStyle(
+                    fontSize: 20.0,
+                  ),
+                ),
+              ),
+            ),
+            Container(
+              height: 50.0,
+              child: Row(
+                  children: <Widget> [
+                    Text(
+                      'Does store exist ?',
+                      style: TextStyle(
+                        fontSize: 20.0,
+                      ),
+                    ),
+                    Switch(
+                      value: isStorePresent,
+                      onChanged: (value) {
+                        setState(() {
+                          isStorePresent = value;
+                          print(isStorePresent);
+                        });
+                      },
+                      activeTrackColor: Colors.lightGreenAccent,
+                      activeColor: Colors.green,
+                      inactiveTrackColor: Colors.red,
+                    ),
+                  ]
+              ),
+            ),
+            Container(
+              height: 50.0,
+              child: Center(
+                child: Row(
+                    children: <Widget> [
+                      Text(
+                        'Does business exist ?',
+                        style: TextStyle(
+                          fontSize: 20.0,
+                        ),
+                      ),
+                      Switch(
+                        value: isBusinessPresent,
+                        onChanged: (value) {
+                          setState(() {
+                            isBusinessPresent= value;
+                            print(isBusinessPresent);
+                          });
+                        },
+                        activeTrackColor: Colors.lightGreenAccent,
+                        activeColor: Colors.green,
+                        inactiveTrackColor: Colors.red,
+                      ),
+                    ]
+                ),
+              ),
+            )
+          ],
+        ),
+      ),
+
+      floatingActionButton: FloatingActionButton.extended(
+        onPressed: (){
+          _showVerifyDialog();
+        },
+        tooltip: 'Verify',
+        shape: RoundedRectangleBorder(),
+        elevation: 0.0,
+        label: Text(
+          'Verify',
+          style: TextStyle(
+              fontSize: 16.0,
+          ),
+        ),
+      ), // This trailing comma makes auto-formatting nicer for build methods.
+    );
+  }
+
+  void _verifyMerchant(){
```

Comment:
https://dart.dev/guides/language/effective-dart/documentation#consider-writing-doc-comments-for-private-apis

Reply:
done